replica/database.cc

```diff
@@ -805,8 +806,11 @@ future<> database::drop_keyspace_on_all_shards(sharded<database>& sharded_db, co
 }
 
 static bool is_system_table(const schema& s) {
-    return s.ks_name() == db::system_keyspace::NAME || s.ks_name() == db::system_distributed_keyspace::NAME
-        || s.ks_name() == db::system_distributed_keyspace::NAME_EVERYWHERE;
+    auto& k = s.ks_name();
+    return k == db::system_keyspace::NAME ||
+        k == db::system_auth_keyspace::NAME ||
```

Comment:
What about `legacy::AUTH_KS`, we don't have to check for it?

Reply:
No if it wasn't there already otherwise we change (break?) current implementation. Currently auth is somehow not fully system table, e.g. admin can do writes to it. While in auth-v2 only auth DML would be able to do writes.